src/exportNameRule.ts

```diff
@@ -64,9 +64,7 @@ export class Rule extends Lint.Rules.AbstractRule {
             return options.ruleArguments[0];
         }
         if (options instanceof Array) {
-            return typeof options[0] === 'object' ?
-                options[0].allow :
-                <string[]><any>options; // MSE version of tslint somehow requires this
```

Comment:
> `MSE version of tslint`

I don't know what MSE is but agreed that it's probably outdated. *M*icro*s*oft... *E*nterprise? 

/cc @HamletDRC in case this is relevant?